infra/variables.tf

```diff
@@ -13,6 +13,16 @@ variable "chat_model_version" {
   default = "2024-08-06"
 }
 
+variable "emb_model_name" {
```

Comment:
I would call this `embedding_model_name` to make it easier to search in the code